src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSmall = 20;
+  const discountRegular = 50;
+
+  if (days >= 3 && days < 7) {
+    return 40 * days - discountSmall;
+  }
+
+  if (days >= 7) {
+    return 40 * days - discountRegular;
+  }
+
+  return 40 * days;
 }
 
```

Comment:
we are almost there
let's create a variable for day cost, and basic cost, do not repeat yourself